benchmarks/bare.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 const server = require('http').createServer(function (req, res) {
-  res.setHeader('content-type', 'application/json')
+  res.setHeader('content-type', 'Content-Type: application/json; charset=utf-8')
```

Comment:
I think I'm missing something here, why you are adding the header name inside the header value?

Reply:
Same question as @delvedor here...

Reply:
I just did some bad copy-paste. The actual difference is in the `charset` addition, that's the full content-type that we should be measuring against, as that's what understood by all browsers.